src/netbase.cpp

```diff
@@ -55,11 +55,16 @@ enum Network ParseNetwork(const std::string& net_in) {
 std::string GetNetworkName(enum Network net) {
     switch(net)
     {
+    case NET_UNROUTABLE: return "unroutable";
     case NET_IPV4: return "ipv4";
     case NET_IPV6: return "ipv6";
     case NET_ONION: return "onion";
-    default: return "";
+    case NET_I2P: return "i2p";
+    case NET_CJDNS: return "cjdns";
+    case NET_INTERNAL: return "internal";
+    case NET_MAX: assert(false);
     }
+    assert(false);
```

Comment:
49f7be8396d85b1795c65ca5ebf6069c6c1e047e

Mind following [Developer Notes -- C++ data structures](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#c-data-structures):
```suggestion
    case NET_MAX: assert(false);
    } // no default case, so the compiler can warn about missing cases
    assert(false);
```
?

Reply:
Thanks for reviewing--I don't mind, but I hesitate to invalidate 3 reviews for a non-essential code comment.

Reply:
ofc, it's not a blocker :)

Reply:
Thanks for reviewing--I don't mind, but I hesitate to invalidate 3 reviews for a non-essential code comment. Can add it in another PR.

Reply:
Had to rebase, so updating with your feedback. Thanks!